frontend/src/entry/CreateEntry.tsx

```diff
@@ -60,6 +62,8 @@ export default function CreateEntry({ createEntryKind, onBackClick }: CreateEntr
   const [collaborators, setCollaborators] = useState<CollaboratorEntry[]>(
     currentUser ? [{ entity: `${EntityKind.USER}:${currentUser?.dn}`, roles: ['owner'] }] : [],
   )
+  const [ungovernedAccess, setungovernedAccess] = useState<boolean>(false)
```

Comment:
`boolean` and `string` can be inferred from their defaults so we don't need to declare the types, so we can just do `useState(false)`.